tracer/test/test-applications/integrations/Samples.DataStreams.ManualAPI/Program.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Datadog.Trace;
+
+namespace Samples.DataStreams.ManualAPI;
+
+public class Program
+{
+    private static ConcurrentQueue<string> _queue = new();
+
+    private static async Task Main(string[] args)
+    {
+        var writing = Send("my message");
+        var reading = Receive();
+
+        // wait up to 1 second for threads
+        await Task.WhenAny(Task.WhenAll(reading, writing), Task.Delay(millisecondsDelay: 1000));
```

Comment:
Any reason for this timeout? 🤔 Seems like a recipe for flake...

Reply:
> it should always complete in 200ms max

This is a classic fallacy of timeouts in tests though 😄 There's no guarantee that the 100ms requested is _actually_ 100ms (or even close if it's under load), and there's basically no good reason to risk it - flake is _way_ worse than a slow test. I'd bump it to 10s at the very least, maybe even more (because why not? 🤷‍♂️)

Reply:
okay